app/controllers/spree/admin/resource_controller.rb

```diff
@@ -34,7 +34,7 @@ def update
           flash[:success] = flash_message_for(@object, :successfully_updated)
           redirect_to location_after_save unless request.xhr?
         end
-        format.turbo_stream if turbo_enabled?
+        # format.turbo_stream if turbo_enabled?
```

Comment:
@damianlegawiec What are you thoughts on this, I know it has a method in the protected area of this controller #turbo_enabled? should we remove the accompanying method and also remove this line, or look into restoring this and find out why the tests were failing on this?

Reply:
@MatthewKennedy @rafalkosla101 yeah, we need this back - we can temp skip tests but we really need this back

Reply:
@damianlegawiec Looking at this in my code editor this is only commented out in the create action but it is still used in the update action.

Maybe removing it from the create action might not be so bad as streams are normal for updates, or it could be more specific like:

# turbo_stream_on_update_enabled?
# turbo_stream_on_create_enabled?

Reply:
Tests are failing on it because it takes enormous time to load the page when turbo is enabled(it is like 15s on a local machine) - it is a bug I believe